e2e/integration/functional/SearchPage.js

```diff
@@ -4,13 +4,19 @@ import els from '../../elements/de';
 
 describe('function tests search page', () => {
   it('should search with fitting products', () => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
This is actually caused by the frontend code, so it should be rather fixed than ignored ;)

![screen shot 2018-08-02 at 16 48 14](https://user-images.githubusercontent.com/3393569/43591592-f624cf30-9673-11e8-98e5-538f3ddbf5d6.png)

Reply:
I'll push a fix for that. + removal of  this workaround.
@fkloes could you check my last commit if I didn't break something?

Reply:
![screen shot 2018-08-02 at 17 06 39](https://user-images.githubusercontent.com/3393569/43592704-78a95e7e-9676-11e8-9208-82fa0be186e3.png)
This fails way to often. Should be fixed somehow. Maybe some timeout or double check?